Ex_22/index.js

```diff
@@ -0,0 +1,13 @@
+"use striect"
+alert ('Hello, user!');
+let firstNum = Number(prompt ('Please, enter first number', 100));
+let secondNum = Number(prompt ('Please, enter second number', 100));
+
+if (firstNum > secondNum){
+    alert('The first number is greater than second number');
+}
+else if (firstNum < secondNum) {
+      alert ('The first number is less then secong number!');
+}
+else {alert('The numbers are the same!')};
```

Comment:
Even though it's one line, still better to write it like this. For code consistency and readability,
`else {
    alert('some text');
}`